src/lib/actions/movies.ts

```diff
@@ -190,6 +191,7 @@ export const addOrUpdateReaction = async (
       return { message: "Reaction added" };
     }
   } catch (e) {
+    console.error("Error adding/updating reaction:", e);
```

Comment:
nie wiem czemu, ale czasem jak testowałem to taki sposób logowania błędu crashował mi aplikacje, może daj JSON.stringify(e) dla pewności